src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -8019,12 +8046,15 @@ namespace SurfaceGeometry {
 					kivaManager.settings.deepGroundBoundary = HeatBalanceKivaManager::KivaManager::Settings::ZERO_FLUX;
 				} else if (SameString(cAlphaArgs( alpF ), "GroundWater")) {
 					kivaManager.settings.deepGroundBoundary = HeatBalanceKivaManager::KivaManager::Settings::GROUNDWATER;
-				} else /* if (SameString(cAlphaArgs( alpF ), "Autoselect")) */ {
+				} else if (SameString(cAlphaArgs( alpF ), "Autoselect")) {
```

Comment:
I know it wasn't added in this PR, but why "Autoselect" over an existing term like "Autocalculate", especially when the IDFs below had "Autocalculate" in the field?